components/connectivity-adapter/internal/connectorservice/api/certificates.go

```diff
@@ -48,10 +48,8 @@ func (ch *certificatesHandler) SignCSR(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	contextLogger.Info("Generating certificate")
```

Comment:
I think we should not remove this log entry.

Reply:
Ok, fixed. My mistake